src/fides/api/ops/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -230,3 +245,97 @@ def save_privacy_preferences(
     consent_request.save(db=db)
 
     return upserted_current_preferences
+
+
+@router.get(
+    CURRENT_PRIVACY_PREFERENCES,
+    status_code=HTTP_200_OK,
+    dependencies=[
+        Security(verify_oauth_client, scopes=[CURRENT_PRIVACY_PREFERENCE_READ])
+    ],
+    response_model=Page[CurrentPrivacyPreferenceReportingSchema],
+)
+def get_current_privacy_preferences(
+    *,
+    params: Params = Depends(),
+    db: Session = Depends(get_db),
+    created_lt: Optional[datetime] = None,
+    created_gt: Optional[datetime] = None,
+) -> AbstractPage[CurrentPrivacyPreference]:
+    """Returns the most recently saved privacy preferences for a given privacy notice"""
+
+    validate_start_and_end_filters([(created_lt, created_gt, "created")])
+
+    query: Query[CurrentPrivacyPreference] = db.query(CurrentPrivacyPreference)
+
+    if created_lt:
+        query = query.filter(CurrentPrivacyPreference.created_at < created_lt)
+    if created_gt:
+        query = query.filter(CurrentPrivacyPreference.created_at > created_gt)
+
+    query = query.order_by(CurrentPrivacyPreference.created_at.desc())
+
+    return paginate(query, params)
+
+
+@router.get(
+    HISTORICAL_PRIVACY_PREFERENCES,
+    status_code=HTTP_200_OK,
+    dependencies=[
+        Security(verify_oauth_client, scopes=[PRIVACY_PREFERENCE_HISTORY_READ])
+    ],
+    response_model=Page[ConsentReportingSchema],
+)
+def get_historical_consent_reporting(
+    *,
+    params: Params = Depends(),
+    db: Session = Depends(get_db),
+    request_timestamp_gt: Optional[datetime] = None,
+    request_timestamp_lt: Optional[datetime] = None,
```

Comment:
This is just a first pass of consent reporting.  We will have more fields to add that we start collecting about a user outside of the privacy center, plus more filtering, CSV export, etc.

I've reticketed these other items here: https://github.com/ethyca/fides/issues/3069